auditor.js

```diff
@@ -15,12 +15,18 @@
  */
 'use strict';
 
-module.exports = function(audits) {
-  return function audit(results) {
-    var flattenedAudits = results.reduce(function(prev, curr) {
+class Auditor {
+
+  flattenArtifacts_(artifacts) {
```

Comment:
https://google.github.io/styleguide/javascriptguide.xml#Visibility__private_and_protected_fields_ indicates that private and protected are suffixed not prefixed :-/